python/cudf/cudf/core/dataframe.py

```diff
@@ -1300,6 +1300,9 @@ def _repr_latex_(self):
     # unary, binary, rbinary, orderedcompare, unorderedcompare
     def _apply_op(self, fn, other=None, fill_value=None):
 
+        if isinstance(other, np.ndarray) and other.ndim == 0:
+            other = other.item()
```

Comment:
Can you put this into the other case statement? Should be able to do something like this and then recursively call `_apply_op`

Reply:
I have added it along the line with other scalar case, I couldn't merge both since I have to fetch value from array and I don't want to add one more condition inside and due to style can't update `other` inside if condition.

```
        elif isinstance(other, (numbers.Number, cudf.Scalar)):
             for col in self._data:
                 result[col] = op(self[col], other)
+        elif isinstance(other, np.ndarray) and other.ndim == 0:
+            other = other.item()
+            for col in self._data:
+                result[col] = op(self[col], other)
         else:

```